Plugins/SequencePlugin/Source/SequencePlugin/Private/SequenceRPCManager.h

```diff
@@ -46,7 +46,7 @@ class SEQUENCEPLUGIN_API USequenceRPCManager : public UObject
 	 */
 	bool PreserveSessionWallet = false;
 	
-	inline const static FString WaaSVersion = "1.0.0";
+	inline const static FString WaaSVersion = "1.0.0 (Unreal 1.4)";
```

Comment:
Is there a way we can include this where we don't always have to remember to update it?

If not, we should create a release checklist or something 🤔

Reply:
Hey @BellringerQuinn, I performed another push to this branch so that on this line, rather than returning a hardcoded version, it returns the current version from the plugin itself.  What do you think?